setup.py

```diff
@@ -50,7 +50,7 @@
 ]
 
 install_requires = [
-    'mosaicml[libcloud,wandb,oci,gcs]>=0.17.2,<0.18',
+    'mosaicml[libcloud,wandb,oci,gcs]>=0.18.1,<0.19',
```

Comment:
there are a few more places you need to bump the version in the extra deps

Reply:
yep, caught in the test failures 👍